mmv1/templates/terraform/examples/network_security_tls_inspection_policy_custom.tf.erb

```diff
@@ -0,0 +1,136 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_privateca_ca_pool" "default" {
+  provider = google-beta
+  name      = "<%= ctx[:vars]['privateca_ca_pool_name'] %>"
+  location  = "us-central1"
+  tier     = "DEVOPS"
+  publishing_options {
+    publish_ca_cert = false
+    publish_crl = false
+  }
+  issuance_policy {
+    maximum_lifetime = "1209600s"
+    baseline_values {
+      ca_options {
+        is_ca = false
+      }
+      key_usage {
+        base_key_usage {}
+        extended_key_usage {
+          server_auth = true
+        }
+      }
+    }
+  }
+}
+
```

Comment:
we should leave only one line between resource declarations

Reply:
done